universal/spec.yml

```diff
@@ -37,8 +37,45 @@ attrs:
 
     - name: input_dim
       description: input dimension for user supplied dimension. 
-      type: ndarray 
-    
+      type: tuple 
+
+# Dense layer
+name: Dense
+attrs:
+    - name: input_dim
+      description:the input dimensionality 
+      type:tuple
+
+    - name: num_units # similar to output_dim
+      description: the number of neurons in the dense layer
+      type: integer
+
+    - name: init
+      description: weights initialization type
+      type: string
+
+# Activation layer
+# This is a generic activation layer that translates in to most of the activation types
+name: Activation
+attrs:
+    - name: activation
+      description: the name of the activation type
+      type: enum
+      values:
+          - HardShrink
```

Comment:
I added some values here from torch as an example basically listing the supported activation functions. Thoughts?